src/luxonis_ml/data/dataset.py

```diff
@@ -125,16 +146,9 @@ def __init__(
         dataset_id,
         team_name=None,
         dataset_name=None,
-        bucket_type="local",
-        override_bucket_type=False,
+        bucket_type=BucketType.EXTERNAL,
```

Comment:
How do external + local work together?

Reply:
Yeah I agree, if local it shouldn't really matter if internal or external. I'd still default to internal. We'll use the external flag to mark that we are not the owners. Which will be for BYOB.